build/yaml/deployBotResources/js/evaluateDependenciesVariables.yml

```diff
@@ -49,18 +55,16 @@ steps:
             }
           }
           STABLE {
-            if ("${{ parameters.botType }}" -in "Host", "Skill") {
+            if ($botType -in "Host", "Skill") {
               $PackageList = npm show botbuilder@* version | Out-String;
             }
-            elseif ("${{ parameters.botType }}" -in "SkillV3") {
+            elseif ($botType -in "SkillV3") {
               $PackageList = npm show botbuilder@3.* version | Out-String;
             }
             $versionNumber = ($PackageList.Split(" ")[-1]).Trim().TrimStart("'").TrimEnd("'");
           }
-          default { $versionNumber = "${{ parameters.version }}" }
         }
-        Write-Host "Version Number: $versionNumber"
```

Comment:
This shouldn't be removed, it shows the source of the registry in the pipeline's output

Reply:
Agree, re added in a new commit